test/basic-querying.test.js

```diff
@@ -620,6 +620,172 @@ describe('basic-querying', function () {
   });
 });
 
+describe('queries', function() {
```

Comment:
When you run the test, it forms the BDD sentence. It is purposely left generic because this test covers a variety of different query types.